pennylane/io/io.py

```diff
@@ -613,6 +618,125 @@ def circuit(x):
     return plugin_converter(quantum_circuit, measurements=measurements)
 
 
+def to_openqasm(
+    qnode,
+    wires: Optional[WiresLike] = None,
+    rotations: bool = True,
+    measure_all: bool = True,
+    precision: Optional[int] = None,
+) -> Callable[[Any], str]:
+    """Serialize the circuit as an OpenQASM 2.0 program.
+
+    Measurements are assumed to be performed on all qubits in the computational basis.
+    An optional ``rotations`` argument can be provided so that output of the OpenQASM circuit
+    is diagonal in the eigenbasis of the quantum script's observables.
+    The measurement outputs can be restricted to only those specified in the script by setting ``measure_all=False``.
+
+    Args:
+        wires (Wires or None): the wires to use when serializing the circuit.
+            Defaults to ``None``, such that all the wires are used for serialization
+        rotations (bool): in addition to serializing user-specified gates, also include the gates that
+            diagonalize the measured wires such that they are in the eigenbasis of the circuit observables.
```

Comment:
```suggestion
            diagonalize the measured wires such that they are in the eigenbasis of the circuit observables
```
Just for the sake of consistency, since the other entries don't have a period at the end : )